src/components/RightSidebar.jsx

```diff
@@ -20,7 +28,8 @@ export default class RightSidebar extends Component {
               <a href="#">Rainbow</a>
             </Sidebar>
             <Sidebar header="Recent Tweets">
-              <p>Nothing here yet...</p>
+                <a className="twitter-timeline" data-theme="dark" data-height="1080" data-link-color="#FAB81E"
+                   href="https://twitter.com/StevenAJC?ref_src=twsrc%5Etfw">Tweets by StevenAJC</a>
```

Comment:
Could we use the convention where a component/element has multiple attributes, each one is displayed on a new line? For example:

```
<a className="twitter-timeline"
   data-theme="dark"
   ...
>
    Twetts by StevenAJC
</a>
```